### ISSUE TITLE
User Story - Create ML Performance Figures and Evaluation pkl

### ISSUE THREAD

**Issue by TomLiDev** [TomLiDev/mildew-detector#26]
As a **Developer** I want to **Create a function which captures the performance and evolution of the ML through its training** to **Visualise and demonstrate how accuracy has changed over the training epochs**.

- #3 

Acceptance Criteria:

- [x] Create a function which plots the loss function of the ML system over its traing epochs
- [x] Create a function which plots the accuracy of the ML system over its training epochs
- [x] Save figures for use in the dashboard
- [x] Create and save evaluation .pkl file
